formula_renames.json

```diff
@@ -102,8 +102,10 @@
   "postgresql95": "postgresql@9.5",
   "prest": "prestd",
   "presto": "prestodb",
-  "pyqt5": "pyqt",
+  "pyqt5": "pyqt@5",
+  "pyqt6": "pyqt",
   "qt5": "qt@5",
+  "qt6": "qt",
```

Comment:
Why are we adding this? Did we ever have formulae named `pyqt6` and `qt6` that we renamed?

Reply:
Dropped